src/simsopt/field/boozermagneticfield.py

```diff
@@ -361,61 +361,90 @@ def __init__(self, equil, order, mpol=32, ntor=32, N=None, enforce_vacuum=False,
             if self.booz.need_to_run_code:
                 self.booz.run()
 
+        self.asym = self.booz.bx.asym  # Bool for stellarator asymmetry
```

Comment:
AFAIK, everywhere else in the code we store a bool called `stellsym` that is true when symmetry is satisfies. For consistency it would be nice to change this to
```
self.stellsym = not self.booz.bx.asym
```

Reply:
done